### ISSUE TITLE
Dark theme by default?

### ISSUE THREAD

**Issue by morevnaproject** [synfig/synfig#1041]
People love dark theme. People find it looking cool. Dark theme is used by default in many graphic apps. It is better for eyes.

So, how about to make dark theme enabled by default in 1.4.0? ^__^

**Comment by rodolforg** [comment]
Some icons don't have enough contrast in dark mode, I think.

Like that one for swap background - outline colors.

**Comment by ebarranco** [comment]
Yeah, Dark theme no questions ask, but definitely for some icons its readability could be better.
![Screenshot_20191210_163835](https://user-images.githubusercontent.com/11136544/70575660-98ab4300-1b6c-11ea-930e-34bab7be98fc.png)

**Comment by BobSynfig** [comment]
Why not to let the choice to the user during first startup?
Lots of apps and IDE do it ;)

**Comment by ankit-kumar-dwivedi** [comment]
Also we can provide better contrast for icons not visible in the dark mode.
This has to be done someday anyway.

**Comment by morevnaproject** [comment]
> Why not to let the choice to the user during first startup?

I think this will be too much. ^__^ I remember Krita, AfterEffects, Photoshop, DarkTable doing that by default without asking.

> Some icons don't have enough contrast in dark mode

True. Let's identify which ones and fix. We can add bright outline for example.

**Comment by libreartist** [comment]
Hi! I just experimented with the existing Synfig icons, and attempted to create uniformity among them (sorry, lots of Synfig icons belong to different styles). I'm presenting, what I did so far.

![Sample svg](https://user-images.githubusercontent.com/37787543/73364618-e1f85300-42d0-11ea-8f1c-92c89948bc56.png)

Icons are matched against different UI colours. I didn't think any different. I tried to match the new set of icons to match form and colours of existing icons. Along with customising the icons for darker UI, I think it's the right time to touch upon the icons as well.

If Synfig needs a whole new effort for the icons, I want to present the icons (with UI) some of the other apps (just for reference).

**OpenToonz:**
![UI_1](https://user-images.githubusercontent.com/37787543/73365343-246e5f80-42d2-11ea-8a8e-f808f1d520d8.png)

**Affinity Photo:**
![affinity-photo-ss PNG 37d7c35e0f5aff0deeb3220c2bfda893](https://user-images.githubusercontent.com/37787543/73365391-34863f00-42d2-11ea-9597-d7147812def9.png)

These are just references that I love. If wanted, I would like to work on the icons in any of the ways the community expects.

With regards,
Benificiary of Synfig

**Comment by libreartist** [comment]
Adding to the above,
Regarding the icons I did, these are random pick, to see how to bring style uniformity to differently-styled icons. I removed gradients and some of the details, and tweaked them a bit, for the sake of uniformity. If time permits, even the gradients and some details can be added to make them even richer. Also, I tried to follow the **tango icons colour palette**. If any different colour palette has to be used, advise me on that as well.

**Comment by rodolforg** [comment]
@libreartist Fantastic!
I intend to (propose to) change the checkbox for layer visibility in Layer tree dock to an more intuitive icon - and maybe provide another icon for render-enable/disable too instead of itallic. Pretty much like Blender 2.80.
If @morevnaproject aproves this changes, could you please provide those icons?

**Comment by libreartist** [comment]
Sure.

**Comment by morevnaproject** [comment]
Hi! Apologies for delay, I have missed this conversation somehow... 

@libreartist I absolutely love what you did and appreciate your contributions here.

The only thing that made me feel weird are icons of humans, that remind me icons on traphic light, but with opposite coloring ^__^"

![screenshot_008](https://user-images.githubusercontent.com/332868/75610284-870d8200-5b42-11ea-88e6-70d2a312ba03.png)

> I intend to (propose to) change the checkbox for layer visibility in Layer tree dock to an more intuitive icon - and maybe provide another icon for render-enable/disable too instead of itallic. Pretty much like Blender 2.80.

That would be great!

Thanks again!

**Comment by rodolforg** [comment]
@libreartist , as Konstantin approved, could you please provide us icons for these flags:
* layer visibility (visible / hidden)
* layer rendering (enabled / disabled)
* layer selection locking (locked / unlocked)

I'd appreciate it a lot :)

**Comment by morevnaproject** [comment]
> layer selection locking (locked / unlocked)

Just a note:

Layer Selection Locking is actually a Layer Parameter (which means it can be animated).
Other flags you mentioned are not Layer Parameters (they cannot be animated and thus have influence on whole timeline).

Actually, we **_can_** make Layer Selection Locking behave as Layer visibility flags - i.e. not a Layer Parameter and non-animatable. 

What people think about this idea? Does anyone animates Layer Selection Locking and finds that useful?

**Comment by rodolforg** [comment]
@libreartist  ping ? :)

**Comment by libreartist** [comment]
Hi, sorry, I couldn't respond, since I have to relocate, in relation to the recent lockdown situation. Hope, everyone is safe out there. My prayers. I could see that @Agnyy is doing a lot well with the icons over there at #1458. Thank you @morevnaproject @rodolforg for responding.
